site_root/input/resources/examples/Patient-pt-002.json

```diff
@@ -0,0 +1,89 @@
+{
+  "resourceType": "Patient",
+  "id": "pt-002",
+  "meta": {
+    "profile": [
+      "http://fhir.kids-first.io/StructureDefinition/kfdrc-patient"
+    ]
+  },
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-ethnicity",
+      "extension": [
+        {
+          "url": "ombCategory",
+          "valueCoding": {
+            "system": "urn:oid:2.16.840.1.113883.6.238",
+            "code": "2135-2",
+            "display": "Hispanic or Latino"
+          }
+        },
+        {
+          "url": "text",
+          "valueString": "Hispanic or Latino"
+        }
+      ]
+    },
+    {
+      "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-race",
+      "extension": [
+        {
+          "url": "ombCategory",
+          "valueCoding": {
+            "system": "urn:oid:2.16.840.1.113883.6.238",
+            "code": "2106-3",
+            "display": "White"
+          }
+        },
+        {
+          "url": "text",
+          "valueString": "White"
+        }
+      ]
+    },
+    {
+      "url": "http://fhir.kids-first.io/StructureDefinition/species",
+      "valueCodeableConcept": {
+        "coding": [
+          {
+            "system": "http://fhir.kids-first.io/CodeSystem/species",
+            "code": "337915000",
+            "display": "Homo sapiens"
+          }
+        ],
+        "text": "Homo Sapiens"
+      }
+    },
+    {
+      "url": "http://fhir.kids-first.io/StructureDefinition/relation",
+      "extension": [
+        {
+          "url": "subject",
+          "valueReference": {
+            "reference": "Patient/pt-001"
+          }
+        },
+        {
+          "url": "relation",
+          "valueCodeableConcept": {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
```

Comment:
this is a general comment; when I see `CHILD`, `child`, and `Child` all right next to each other, it kind of bugs me

Reply:
@gsantia, good catch! So, if you look into this [ValueSet](https://www.hl7.org/fhir/v3/FamilyMember/vs.html), `code` and `display` are pre-defined. And, if you look into the general description of [CodeableConcept](https://www.hl7.org/fhir/datatypes.html#CodeableConcept), `code` and `display` come from `system` while `text` is a plain, source text description. Say, we have a `"Type 2 diabetes"` diagnosis for a patient. This concept is widely used, but may be expressed in different forms from different ontologies. In SNOMED CT, this is described as `"Diabetes mellitus type 2"` with code `"44054006"`. In HPO, it is described as `"Type II diabetes mellitus"` with code `"HP:0005978"`. In this case, we may want to create the following attribute, using CodeableConcept:

```json
"Condition.code": {
  "coding": [
    {
      "system": "URL of SNOMED CT",
      "code": "44054006",
      "display": "Diabetes mellitus type 2"
    },
    {
      "system": "URL of HPW",
      "code": "HP:0005978",
      "display": "Type II diabetes mellitus"
    },
  ],
  "text": "Type 2 diabetes"
}
```

This way, even though how idiosyncratic a form of a diagnosis is described across multiple systems and by asserter, we can always harmonize multiple expressions referring to the sample concept.